gui/functionality/idact_app.py

```diff
@@ -48,24 +47,16 @@ def __init__(self):
         self.ui.setupUi(self)
 
 
-class ErrorApp(QMainWindow):
-    def __init__(self, message="Error occured"):
-        super(ErrorApp, self).__init__()
-        self.ui = Ui_Error()
-        self.ui.setupUi(self)
-        self.ui.label.setText(message)
-
-
-class SuccessWindow(QWidget):
+class PopUpWindow(QWidget):
     def __init__(self):
         QWidget.__init__(self)
         self.box = QMessageBox(self)
-        self.box.setIcon(QMessageBox.Information)
+    #    self.box.setIcon(QMessageBox.Information)
         self.box.addButton(QMessageBox.Ok)
         self.box.setDefaultButton(QMessageBox.Ok)
-        self.box.setWindowTitle("Success")
 
-    def show_message(self, message):
+    def show_message(self, message, window_title):
```

Comment:
Maybe it is worth to make the `window_title` into some sort of constant(SUCCESS, ERROR, INFO etc.) and then decide what icon we want to show based on it?